contracts/VestingEscrowSimple.vy

```diff
@@ -183,7 +176,7 @@ def apply_transfer_ownership():
     """
     assert msg.sender == self.future_admin  # dev: future admin only
     self.admin = msg.sender
-    self.future_admin = ZERO_ADDRESS
+    self.future_admin = empty(address)
```

Comment:
i don't like this two-part commit thing in this contract, it's not a dao, maybe we should simplify it

Reply:
decided to delete the ability to change admin altogether, based on usage stats (this feature was only used in one escrow) and on the basis of vesting being an agreement between two parties. it makes no sense for one party to be able to swap for someone else.